docs/tutorial/final-app.md

```diff
@@ -223,4 +223,6 @@ button_style = dict(
 
 Congratulations! You have built your first chatbot. From here, you can read through the rest of the documentations to learn about Reflex in more detail. The best way to learn is to build something, so try to build your own app using this as a starting point!
 
+## One More Thing
 
+With our hosting service, you can deploy this app with a single command within minutes. Check out our [Hosting Quick Start](/docs/hosting/deploy-quick-start).
```

Comment:
We should be able to do

```python exec
from reflex.docs import hosting
```

Then reference it as `{hosting.deploy_quick_start}`